pkg/plugins/golang/v4/scaffolds/internal/templates/hack/boilerplate.go

```diff
@@ -106,8 +106,8 @@ Copyright {{ .Year }}.
 {{ index .Licenses .License }}*/`
 
 var knownLicenses = map[string]string{
-	"apache2": apache2,
-	"none":    "",
+	"apache2":   apache2,
+	"copyright": "",
```

Comment:
```suggestion
	"copyright": copyright, 
```

We need to ensure the type, like the above for apache2. Am I right?

Reply:
I think that's right. We can define the const value 'copyright' a empty string. Just like `const copyright = ""`?